app/controllers/models/wallet.js

```diff
@@ -40,6 +40,7 @@ module.exports = function (app, sqlLiteService) {
                         table.string('keystoreFilePath');
                         table.binary('profilePicture');
                         table.integer('isSetupFinished').notNullable().defaultTo(0);
+                        table.integer('isLedger').notNullable().defaultTo(0);
```

Comment:
This should be a string or enum field specifying a `provider` or `profile` which can be any of:

- local (private key stored in wallet file)
- ledger
- trezor (not implemented yet)

Reply:
Okay I will change it to 'enum'

Reply:
enum won't be supported in SQLite - will need to be string